resources/user_config.ts

```diff
@@ -231,7 +264,19 @@ class UserConfig {
             // issues, it's unlikely that these will be able to be anything but eval forever.
             //
             /* eslint-disable no-eval */
-            eval(localFiles[jsFile]);
+            window.eval.call({
+              Conditions,
+              ContentType,
+              NetRegexes,
+              Regexes,
+              Responses,
+              Outputs,
+              Util,
+              ZoneId,
+              ZoneInfo,
+              Options,
+              ...window,
+            }, localFiles[jsFile]);
```

Comment:
Wise thought, but then we should use `this.ZoneId` to reference to those import names?

Reply:
I haven't tested this, I'm not sure it works.